shakedown/dcos/network.py

```diff
@@ -0,0 +1,42 @@
+"""Utilities for working with nodes
+    There are a number of common utilies for working with agents and master nodes
+    which are provided here.
+"""
```

Comment:
These should all be documented in [API.md](https://github.com/dcos/shakedown/blob/master/API.md).